runtime/parachains/src/disputes.rs

```diff
@@ -1152,6 +1152,9 @@ mod tests {
 		while System::block_number() < to {
 			let b = System::block_number();
 			if b != 0 {
+				// circumvent requirement to have bitfields and headers in block for testing purposes
+				crate::paras_inherent::Included::<Test>::set(Some(()));
+
 				AllPallets::on_finalize(b);
```

Comment:
The addition of ParaInherent in the mock `construct_runtime!` causes the ParaInherent `on_finalize` hook to now fire and will panic as no paras inherent have been entered. I believe this to be reasonable for the disputes tests but always good to double check

Reply:
looks fine to me